.gitpod.yml

```diff
@@ -1,8 +1,16 @@
 tasks:
-  - init: mvn install -P quick-build
+  - init: |
+      mvn install -P quick-build
+      curl https://gist.githubusercontent.com/timja/fe73f0a0f98afe4d9356e7ce6752d95b/raw/a8d772267bed81fb0c11af2c69eaa71ebc7daec0/init.sh | bash
```

Comment:
you are better to move it out of [init](https://www.gitpod.io/docs/references/gitpod-yml#tasksninit).

> When you start a workspace that does not have a [prebuild](https://www.gitpod.io/docs/prebuilds), init is executed at workspace start. When you start a workspace that has a prebuild, init executes as part of the prebuild, but **does NOT execute again** at workspace start.

Reply:
```yaml
tasks:
  - init: mvn install -P quick-build
    command: |
      curl https://gist.githubusercontent.com/timja/fe73f0a0f98afe4d9356e7ce6752d95b/raw/a8d772267bed81fb0c11af2c69eaa71ebc7daec0/init.sh | bash
      mvn hpi:run
    name: Run
  - command: gp await-port 8080 && gp url 8080 && gp preview $(gp url 8080)/jenkins/
    name: Preview
```

Reply:
https://github.com/jenkinsci/design-library-plugin/pull/66